### ISSUE TITLE
custom metrics not showing up

### ISSUE THREAD

**Issue by rounders** [librato/librato-rails#118]
I have successfully setup librato-rails and my librato account is receiving the metrics for the various rails suites. However, I haven't had any luck so far in using the custom measurements. I have added the following code to my site:

Librato.increment 'photos.created' 

But I don't see that data anywhere in my librato account. Do I need to add something extra to the 'suites' configuration for the custom measurements to appear in my account?

**Comment by bdehamer** [comment]
@rounders, the suites configuration should have no impact on your custom measurements. I might try bumping up the log level (see https://github.com/librato/librato-rack#troubleshooting) to see if the metric is even being submitted.

Beyond that, you may want to shoot an email to support@ or open a ticket from inside the app in order to get the best help for your issue.

**Comment by rounders** [comment]
thanks @bdehamer

**Comment by rounders** [comment]
@bdehamer I set the log level to trace so I could see exactly what is being sent to Librato and my custom metric is not getting sent. I fired up a rails console and ran Librato.increment 'photos.created' a bunch of times and that metric never gets sent over.

**Comment by bdehamer** [comment]
@rounders be careful trying to test from the Rails console, I believe that the reporter doesn't start when Rails console is started (https://github.com/librato/librato-rails#console-mode).

I just spun-up a new Rails app and added the _librato-rails_ gem along with a _librato.yml_ file containing my creds. I put a call to `Librato.increment 'posts.created'` in one of my controller actions. When looking at the log (_trace_ log level). I see the following:

```
[librato-rails] flushing pid 4276 (2016-05-22 10:38:35 -0700)..
[librato-rails] Queued: {:gauges=>
  [{:name=>"rails.sql.queries", :value=>28},
   {:name=>"rails.sql.selects", :value=>11},
   {:name=>"posts.created", :value=>5},
   {:value=>5,
    :source=>"posts:index.html.erb",
    :name=>"rails.view.render_template"},
...
```

The metric for my `increment` call is definitely in the data being submitted to Librato.

Not entirely sure what's going on in your case, but the `increment` call should work without any additional configuration.

**Comment by rounders** [comment]
@bdehamer I think I know what's going on in my case. My most recent testing was done with the console and as you pointed out, the reporter doesn't start when the console is used. I had previously read that but had unfortunately forgotten about it when I ran my test yesterday. And in the case of my code base, I just realized that the place in which we run the Librato.increment method always occurs in a sidekiq background job, and as I just re-read, background jobs need to be handled in a different way so I'll investigate that further.

Thanks again for your help and pointing me in the right direction 👍
